requirements.txt

```diff
@@ -9,8 +9,9 @@ black[jupyter]
 datasets
 fire
 git+https://github.com/huggingface/peft.git
-transformers>=4.31.0
+git+https://github.com/huggingface/transformers.git@main
```

Comment:
Why are we going from release pypi to rolling release github here?

Reply:
The fast code llama tokenizer wasn't working correctly and only the git version had the fix at the time of this change, hopefully it can be rolled back now